src/current/_includes/v23.2/cdc/disable-replication-ttl.md

```diff
@@ -0,0 +1,4 @@
+{% include_cached new-in.html version="v23.2" %} Use one of the following to prevent changefeeds from sending messages for changes caused by row-level TTL jobs:
+
+- `sql.ttl.changefeed_replication.disabled` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}): when `true`, deletes issued by TTL jobs on the cluster will not be emitted by changefeeds.
+- `disable_changefeed_replication` [session variable]({% link {{ page.version.version }}/set-vars.md %}): when `true`, changefeeds will not emit messages for any changes (e.g., `INSERT`, `UPDATE`) issued to watched tables during that session.
```

Comment:
This new description looks good! I suspect we'll want to move it to a different (non-TTL) page though?

Reply:
This still needs to be removed right?